uploadChromiumBuildToCloudDevelopment.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# Exit on any error
+set -e
+
+echo Uploading chromium build from local dir to cloud-dev
+echo
+
+# Always use ReplayDevelopment aws profile, no matter what the
```

Comment:
Unfortunately not everyone has named their profiles the same thing but this is probably fine for now.